testcafe/tests/drive/init-data.js

```diff
@@ -28,6 +29,7 @@ const TEST_PREPARE = `1-2 Prepare the unzipped files for testing`
 
 const FIXTURE_INIT = `${FEATURE_PREFIX} 2- Upload Data`
 const TEST_CREATE_FOLDER = `2-1 Create Folder & Upload Files`
+const TEST_CONFLICT_UPLOAD = `2-2 Upload Files already on drive`
```

Comment:
It feels "strange" that the test lives in `init-data.js`
Maybe rename it to "Upload a file that is already on drive" ? (since it only uploads one file)